src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -160,22 +188,24 @@ class WorkspaceNewRoomPage extends React.Component {
                                 label={this.props.translate('workspace.common.workspace')}
                                 placeholder={{value: '', label: this.props.translate('newRoomPage.selectAWorkspace')}}
                                 items={this.state.workspaceOptions}
-                                onChange={this.onWorkspaceSelect}
+                                errorText={this.state.errors.policyID}
+                                onChange={policyID => this.clearErrorAndSetValue('policyID', policyID)}
                             />
                         </View>
                         <Picker
                             value={this.state.visibility}
                             label={this.props.translate('newRoomPage.visibility')}
                             items={visibilityOptions}
-                            onChange={visibility => this.setState({visibility})}
+                            errorText={this.state.errors.visibility}
```

Comment:
```suggestion
```

This error should never happen as a default visibility is always set.

Reply:
Sure. Thanks.